libpages/config/utils.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package config
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+)
+
+// GenerateSHA256PasswordHash generates a sha256 hashed password hash from
+// plaintextPassword in the form of "sha256:<hash>". This is what should go
+// into the config file.
+func GenerateSHA256PasswordHash(plaintextPassword string) string {
+	hash := sha256.Sum256([]byte(plaintextPassword))
```

Comment:
Would it be good practice to add a nonce in here (that's also stored in the config), so it's harder to tell when passwords are being reused?

Reply:
Wait actually, what do we get from making it harder to tell when passwords are reused?

Reply:
It just seems like the right thing to do from a security standpoint.  Obviously Keybase still has to handle the plaintext passwords because that's how http auth works.  But you could imagine someone putting a config file in a public or team folder, but not wanting to expose the fact that two users have the same password to other readers of that directory.

Reply:
That's a good case. Will do!